test/session_test.exs

```diff
@@ -39,6 +38,7 @@ defmodule Janus.SessionTest do
       assert_receive {:trace, ^conn, :receive, %{"janus" => "ack"}}, 2 * interval
     end
 
+    @moduletag :capture_log
```

Comment:
What does this do?

Reply:
Hides log of exiting process, we enforce process exit so there is no need to show it during tests.

Reply:
@moduletag should be changed to @tag